prow/scripts/library.sh

```diff
@@ -271,6 +290,10 @@ function gkeCleanup() {
         "${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}"/delete-dns-record.sh --project="${CLOUDSDK_CORE_PROJECT}" --zone="${CLOUDSDK_DNS_ZONE_NAME}" --name="${APISERVER_DNS_FULL_NAME}" --address="${APISERVER_IP_ADDRESS}" --dryRun=false
     fi
 
+    if [ -n "${TMP_DIR}" ]; then
+        rm -rf "${TMP_DIR}"
+    fi
```

Comment:
Why do we want to remove the `TMP_DIR` if it's going to be removed anyway after the job is finished?

Reply:
I found it in the [kyma-gke-upgrade.sh](https://github.com/kyma-project/test-infra/blob/master/prow/scripts/cluster-integration/kyma-gke-upgrade.sh#L160) script. I didn't want to break anything so I preserved it here. If it's not needed I can delete it

Reply:
Looks like an unnecessary step, as the temporary directories are not stored anywhere after the prowjob is done.

Reply:
Ok! I deleted it then 🎉